### PR TITLE
Bundle links

### PR DESCRIPTION
**Pull request by huwshimi**
Add source and bug links to bundle details.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the details panel for a bundle
- check that there are Source and Bundle links in the header and they go to the correct locations

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2363/
Test FAILed.

**Comment by makyo** [comment]
QA issue: it appears to go to the bundle name, rather than the basket name for bug links (i.e: it links to https://bugs.launchpad.net/charms/+source/mediawiki-single but should link to https://bugs.launchpad.net/charms/+source/mediawiki), so probably some sort of lp: munging like you do for the source link is in order.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2365/
Test FAILed.

**Comment by makyo** [comment]
:+1: I'd like to see a test around the bundles stuff in case that changes later (a test that breaks with a change is appreciated).  Thanks!

**Comment by huwshimi** [comment]
Thanks @makyo I've added a test.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2366/
Test PASSed.

File: app/subapps/browser/views/bundle.js [hunk 1]
```diff
@@ -193,13 +193,15 @@ YUI.add('subapp-browser-bundleview', function(Y) {
...
       var templateData = bundle.getAttrs();
+      var sourceLocation = templateData.code_source.location;
       templateData.charmIcons = utils.charmIconParser(templateData.services);
       // Remove the svg files from the file list
       templateData.files = templateData.files.filter(function(fileName) {
         return !/\.svg$/.test(fileName);
       });
       templateData.services = this._buildCharmList(templateData.services);
-      templateData.sourceLink = templateData.code_source.location;
+      templateData.sourceLink = this._getSourceLink(sourceLocation);
```
Comment (hatched) [inline app/subapps/browser/views/bundle.js]: Where is this defined?

Comment (hatched) [inline app/subapps/browser/views/bundle.js]: nm :)

**Comment by hatched** [comment]
:+1: !

**Comment by huwshimi** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/823

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**